config/opal_setup_java.m4

```diff
@@ -113,8 +113,9 @@ AC_DEFUN([OPAL_SETUP_JAVA],[
                           with_jdk_headers=$opal_java_dir/include
                           with_jdk_bindir=$opal_java_dir/bin
                       else
-                          AC_MSG_WARN([No recognized directory structure found under $opal_java_dir])
-                          AC_MSG_ERROR([Cannot continue])
+                          AC_MSG_WARN([No recognized OS X/macOS JDK directory structure found under $opal_java_dir])
+                          opal_java_found=0
+                          AC_MSG_RESULT([Falling back to other platforms' JDK checks])
```

Comment:
Good catch -- this does seem like it was the error (from my original pseudocode in a prior github comment).

I think you can get rid of the AC_MSG_RESULT -- it'll be obvious in the configure output that we're trying other places.

Reply:
@jsquyres:  Just thought I'd be consistent with there being a message there before (albeit the faulty one,) but I'll change it.